hadoop-ozone/insight/src/main/java/org/apache/hadoop/ozone/insight/ConfigurationSubCommand.java

```diff
@@ -51,16 +51,31 @@ public Void call() throws Exception {
         "Configuration for `" + insightName + "` (" + insight.getDescription()
             + ")");
     System.out.println();
-    for (Class clazz : insight.getConfigurationClasses()) {
-      showConfig(clazz);
 
+    Type type;
+    switch (insightName.split("\\.")[0]) {
```

Comment:
How about change the switch expression to the following way?

```java
try {
  Type type =Type.valueOf(insightName.split("\\.")[0]);
} catch (IllegalArgumentException e) {
   throw new RuntimeException("No such component " + insightName);
}
```

Reply:
Thanks @maobaolong for the suggestion.

I found we wouldn't catch the ```IllegalArgumentException``` that would be detected in L49 ```getInsight()``` if it is illegal.
Rest fixed.